docs/high_availability_installation_guide.md

```diff
@@ -0,0 +1,284 @@
+# Harbor High Availability Guide 
+
+## Abstract 
+
+This guide describes how to install and configure Harbor for high availability. It supplements the Installation Guides and assumes that you are familiar with the material in those guides. 
+
+**Important!** 
+
+This guide was last updated as of the Harbor 1.4 release. It will not apply to releases before 1.4. We advise that you read this at your own discretion when planning on your Harbor high availability implementation.  
+
+This guide is intended as advice only. It doesn't support Notary(Content trust functionality) for now. 
+
+## Contents 
+
+**Notices** 
+
+Notices take these forms: 
+
+**Note!**: A comment with additional information that explains a part of the text. 
+
+**Important!**: Something you must be aware of before proceeding. 
+
+**Tip!**: An extra but helpful piece of practical advice. 
+
+**Caution!**: Helpful information that prevents the user from making mistakes. 
+
+**Warning!**: Critical information about the risk of data loss or security issues. 
+
+## Harbor High Availability Introduction 
+
+This document discusses some common methods of implementing highly available systems, with an emphasis on the core Harbor services and other open source services that are closely aligned with Harbor. 
+
+You will need to address high availability concerns for any applications software that you run on your Harbor environment. The important thing is to make sure that your services are redundant and available. How you achieve that is up to you. 
+
+### Stateless service 
+To make a stateless service highly available, you need to provide redundant instances and load balance them. Harbor services that are stateless include: 
+ - Adminserver
+ - UI
+ - Registry
+ - Logs
+ - Jobservice
+ - Clair
+ - Proxy 
+
+### Stateful service 
+
+Stateful services are more difficult to manage. Providing additional instances and load balancing does not solve the problem. Harbor services that are stateful include the follow services:
```

Comment:
"Providing additional instances and load balancing does not solve the problem" I think this statement is not correct. DB can also be scaled and HA.

Reply:
This is a common issue for statefull services.  If you simply build a  load balance on top of statefull services with out setting up back-end replication will not works. You still need to do extra works beside the simply add instance and use LB.